index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+  <title>Easy Science</title>
+  <script src="script.js" defer></script>
+</head>
+<body>
+  <header>
+    <section class="social-network">
+      <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">Español</a></li>
+      </ul>
+    </section>
+    <nav>
+      <div class="logo"><a href="index.html"><strong>E</strong>asy <strong>S</strong>ience</a></div>
+      <div class="hamburger-menu"><img id="menu-button" src="assets/hamburger-menu.png" alt="menu"></div>
+      <ul id="menu">
+        <li class="closeButtonContainer"><span id="closeButton">X</span></li>
+        <li><a class="Personal-information" href="courses.html">Courses</a></li>
+        <li><a class="Personal-information" href="#about">About</a></li>
+        <li><a class="Personal-information" href="#contact">Contact</a></li>
+        <li><a class="Personal-information button sign-up" href="#">Sign up</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <section class="home-page">
+        <div class="home-text">
+          <h1>We teach science the right way</h1>
+          <p>We have the mission to provide a world-class education for anyone, anywhere.</p>
+          <a class="button" href="courses.html">Start here</a>
+        </div>
+    </section>
+    <section id="about" class="about">
+      <h2>Find a comprehensive explanation for the subject you've been struggling with</h2>
+      <hr>
+      <div>
+        <p>Start with:</p>
+        <ul>
+          <li class="button"><a href="#">Math   <i class="fas fa-square-root-alt"></i></a></li>
+          <li class="button"><a href="#">Physics   <i class="fas fa-atom"></i></a></li>
+          <li class="button"><a href="#">Chemistry   <i class="fas fa-flask"></i></a></li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ] [REQUIRED] The middle section too is beautiful in your concept, but the project required a column that includes a section header, a divider, five individual articles, each with an image, title, and description, and a line of text at the bottom.
- [ ] Let's also make sure the design pattern below is followed according to the requirements of the project not forgetting the mobile view requirements too

![image](https://user-images.githubusercontent.com/4247740/147304759-67745c92-d6e0-4a3e-9f32-c9a9b266c51f.png)